.github/workflows/p2p-execute-command.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
     env:
       env_vars: ${{ secrets.env_vars }}
       BASE_DOMAIN: ${{ vars.BASE_DOMAIN }}
+      INTERNAL_DOMAIN: ${{ vars.INTERNAL_DOMAIN }}
```

Comment:
should it be `INTERNAL_BASE DOMAIN`?

also thinking if we even need to define both as we'll have the construction:
*<env>.<domain>
*<env>-internal.<domain>

Could we perhaps just define a domain and construct the rest? would need to think how this can be non breaking though

Reply:
I see they call it `internal_services_domain` or `internalServicesDomain` in terraform files. https://github.com/coreeng/developer-platform/blob/main/features/platform-monitoring/terragrunt-aws.hcl#L10

Maybe I should call it `INTERNAL_SERVICES_DOMAIN` as well?

Resolving it from the base domain should be doable, but I can't really decide here, maybe there are some valid reasons to have it as a separate parameter.